### DISCUSSION TITLE
More verbose logs

### DISCUSSION THREAD

**Discussion by Lionel-dev** [Budibase/budibase#11462] [Help]
Hello, 

I'm currently having several relational problems with my MySQL DB. Unfortunately, when I make queries in Workbench, everything works, but when I try to request the same thing in Budibase, it doesn't show me all the data. So I'd like to find out what Budibase is running to analyze what's wrong with my DB structure so that it can't get my data out... 

Ex:
 I request all rows ASC limit 5 --> Ok
 I request all rows DESC limit 5 --> KO I only have 3... 
 
 If I make these requests in workbench on the same table, it works perfectly. Thanks for your answers :)

**Comment by andz-bb** [comment]
hey @Lionel-dev, could you share your MySQL [DB schema](https://docs.budibase.com/docs/troubleshooting-sql#sql-schema-export) and an [app export](https://docs.budibase.com/docs/export-and-import-apps#export-an-app) so that I can investigate this for you on my end? Thanks
